### PR TITLE
native automation: disable multiple windows if native automation is enabled

### PR DESCRIPTION
**Pull request by AlexKamaev**

### CODE REVIEW

**Comment by AlexKamaev** [comment]
@titerman check the error message please

File: src/client/driver/driver.js [hunk 1]
```diff
@@ -411,6 +413,17 @@ export default class Driver extends serviceUtils.EventEmitter {
             });
     }
 
+    _onBeforeChildWindowOpened (e) {
+        if (this.options.nativeAutomation) {
```
Comment (miherlosev) [inline src/client/driver/driver.js:417]: Remove the one level of curly brackets.

```js
if (!this.options.nativeAutomation)
    return;

....
```

File: test/functional/fixtures/regression/gh-7599/test.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const expect                     = require('chai').expect;
```
Comment (miherlosev) [inline test/functional/fixtures/regression/gh-7599/test.js:1]:
```js
const { expect } = require('chai');
```

**Review by miherlosev — CHANGES_REQUESTED** [review]

**Review by miherlosev — APPROVED** [review]

File: src/errors/test-run/templates.js [hunk 3]
```diff
@@ -406,6 +406,10 @@ export default {
         Multi-window mode is supported only in locally-installed Chrome, Chromium, Edge 84+ and Firefox. Run tests in these browsers to use the "${err.methodName}" method.
     `,
 
+    [TEST_RUN_ERRORS.multipleWindowsModeIsNotSupportedInNativeAutomationError]: () => `
+        Multi-window mode is not supported in the Native Automation mode. To run test in multiple windows, please remove the "--native-automation" option.
```
Comment (titerman) [inline src/errors/test-run/templates.js:410]:
```suggestion
        The Native Automation mode does not support the use of multiple browser windows. Disable the "native automation" option to continue.
```

**Pull request merged**